ci-base/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG CI_COMMON_VERSION=1.0
-ARG ASWF_ORG=aswfstaging
+ARG ASWF_ORG=aswfaswftesting
```

Comment:
Is this the actual org name? Looks like a typo but wanted to check.

Reply:
Good catch, thanks! Just pushed a fix for this silly search and replace issue...